nullaway/src/test/java/com/uber/nullaway/EnsuresNonNullTests.java

```diff
@@ -48,20 +48,16 @@ public void requiresEnsuresNonNullStaticFields() {
             "class Foo {",
             "  @Nullable static Item nullableItem;",
             "  @RequiresNonNull(\"nullableItem\")",
-            "  // BUG: Diagnostic contains: For @RequiresNonNull annotation, cannot find instance field",
             "  public static void run() {",
-            "    // BUG: Diagnostic contains: dereferenced expression nullableItem is @Nullable",
             "    nullableItem.call();",
             "     ",
             "  }",
             "  @RequiresNonNull(\"this.nullableItem\")",
```

Comment:
This is still wrong; we don't want to allow writing `this.nullableItem` to refer to static field `nullableItem`, even though Java allows it.

Reply:
Fixed!